drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/testutil/TestUtil.java

```diff
@@ -103,11 +119,14 @@ public static void assertDoesNotContain(final String notExpected,
                                             final int rowNumber) {
 
         assertFalse("Found " + notExpected,
-                    result.stream().filter(issue -> titleEquals(issue,
-                                                                notExpected) && rowNumberContains(issue,
-                                                                                                    new HashSet<Integer>() {{
-                                                                                                        add(rowNumber);
-                                                                                                    }})).count() > 0);
+                    result.stream()
+                          .filter(issue -> titleEquals(issue,
+                                                       notExpected)
+                                           && rowNumberContains(issue,
+                                                                new HashSet<Integer>() {{
```

Comment:
I was not able to find such method in `Collections`. Isn't this some news in Java 9 or something?